task-manager-backend/src/task/controllers/delete-all-tasks/delete-all-tasks.js

```diff
@@ -0,0 +1,23 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { deleteAllTasksUsecase } = require("../../usecases");
+
+const { TASK_DELETE_SUCCESS } = require("../../../commons/constants");
+
+module.exports = {
+    deleteAllTasks: {
+        path: "/",
+        reqType: "delete",
+        method: async (req, res, next) => {
+            try {
+                const userId = req.session.userId;
+                await deleteAllTasksUsecase.execute(userId);
```

Comment:
here we can directly pass req.session.userId

Reply:
okay